apps/svelte.dev/src/lib/server/renderer.ts

```diff
@@ -7,11 +7,7 @@ export const render_content = (
 ) => {
 	return render_content_markdown(filename, body, options, (filename, source) => {
 		// TODO these are copied from Svelte and SvelteKit - adjust for new filenames
-		const injected = [];
-
-		if (/(svelte)/.test(source) || filename.includes('typescript')) {
```

Comment:
This was previously always being injected for any file that mentions 'svelte'. Instead, we can just reference the Svelte types in the Twoslash typescript options